src/ObjectList.php

```diff
@@ -357,4 +359,19 @@ public function sum(callable $valueProvider, $startValue = null)
             return $total + $valueProvider($item);
         }, $startValue);
     }
+
+    /**
+     * @param callable $filter
+     * @return mixed
+     */
+    public function get(callable $filter)
+    {
+        foreach ($this as $item) {
+            if ($filter($item)) {
+                return $item;
+            }
+        }
+
+        throw new RuntimeException("Could not find an item in " . get_class($this) . " using a find filter.");
```

Comment:
Ok, I get that (lol), but what is the point of adding so strict method into very flexible structure?